client/blocks/post-likes/style.scss

```diff
@@ -3,16 +3,55 @@
 
 	.post-likes__item {
 		display: inline-block;
-		margin: 2px 8px 2px 0;
 		vertical-align: top;
+		text-decoration: none;
+		padding: 3px;
+
 		.gravatar {
 			display: block;
 		}
 	}
 
-	.post-likes__placeholder {
+	&.has-display-names {
+		max-height: 340px;
+		overflow-y: auto;
+
+		.post-likes__item {
+			display: block;
+			position: relative;
+			padding: 4px 16px 4px 8px;
+			border-bottom: 1px solid $gray-light;
```

Comment:
Will we need to account for the last `.post-likes__item` differently when `renderExtraCount` returns `null`?

Reply:
![image](https://user-images.githubusercontent.com/363749/34265045-6d6fba72-e63a-11e7-99fa-23238aa93eb4.png)

It looks like this might be what's causing the rounded corners on the bottom to look weird.